ksql-execution/src/main/java/io/confluent/ksql/execution/expression/tree/StringLiteral.java

```diff
@@ -32,7 +31,7 @@ public StringLiteral(final String value) {
 
   public StringLiteral(final Optional<NodeLocation> location, final String value) {
     super(location);
-    this.value = StringUtil.cleanQuotes(Objects.requireNonNull(value, "value"));
+    this.value = cleanQuotes(Objects.requireNonNull(value, "value"));
```

Comment:
I'm 99.9% sure this isn't actually needed.  There may be some bad tests somewhere that create `StringLiteral`s with quotes, but in the normal path the parser removes the quotes - so this is actually a bug: it can potentially remove escaped quotes that are part of a literal.

Reply:
You are right: 
```java
    @Override
    public Node visitStringLiteral(final SqlBaseParser.StringLiteralContext context) {
      return new StringLiteral(
          getLocation(context),
          ParserUtil.unquote(context.STRING().getText(), "'")
      );
    }
```